### PR TITLE
Notes 

### PR DESCRIPTION
**Pull request by saketkum11**
- note card
- delet notes
- update notes
- editor modal
- color palette
- tags
- Reactquill

### CODE REVIEW

File: src/Context/Auth/Auth-Context.js [hunk 1]
```diff
@@ -29,24 +31,28 @@ const AuthProvider = ({ children }) => {
...
+    try {
+      const response = await axios.post("/api/auth/login", {
+        email: email,
+        password: password,
+      });
+      localStorage.setItem("token", response.data.encodedToken);
+      navigate("/notes");
+    } catch (error) {
+      console.error(error);
     }
```
Comment (raghav6202) [inline src/Context/Auth/Auth-Context.js:43]: Can remove extra space here

**Review by raghav6202 — APPROVED** [review]
Code looks clean. File structure is good too.

File: src/Context/Auth/Auth-Context.js (hunk 1 above)
Comment (saketkum11) [reply src/Context/Auth/Auth-Context.js:43]: Yaa sure .

File: src/Component/Card-note/CardNote.jsx [hunk 2]
```diff
@@ -0,0 +1,53 @@
+import { useNote } from "../../Context/Note/note-context";
+import { useState } from "react";
+import "../../Style/cardNote.css";
+import Typographycal from "../Typographcal/Typographycal";
+import Palette from "../Palette/Palette";
+import Tags from "../Tags/Tags";
+import Priority from "../Priority/Priority";
```
Comment (MiheerTamkhane) [inline src/Component/Card-note/CardNote.jsx:7]: Use Barrel exports here.

File: src/Component/Card-note/CardNote.jsx [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+              placeholder="Title"
+              onChange={(e) =>
+                noteDispatch({ type: SET_TITLE, payload: e.target.value })
+              }
+            />
+            <button className="text-color-5 bg-black-0 rounded-full outline-none border-none pd-x-3 pd-y-2 cursor">
+              <i className="fa-solid fa-thumbtack"></i>
+            </button>
+          </div>
+          <Typographycal></Typographycal>
```
Comment (MiheerTamkhane) [inline src/Component/Card-note/CardNote.jsx:30]: You can use self closing tag like : <Typographycal />

File: src/Component/Card-note/CardNote.jsx (hunk 2 above)
Comment (saketkum11) [reply src/Component/Card-note/CardNote.jsx:7]: yes I will use this method .
thank you.

Comment (saketkum11) [reply src/Component/Card-note/CardNote.jsx:7]: OkK forgot to do that.

File: src/Component/EditorModal/EditorModal.jsx [hunk 4]
```diff
@@ -0,0 +1,209 @@
+import React, { useState } from "react";
+import "../EditorModal/EditorModal.css";
+import ReactQuill from "react-quill";
+import "react-quill/dist/quill.snow.css";
+import { useNote } from "../../Context/Note/note-context";
+
+function EditorModal({ note }) {
+  const { _id, noteText, color, noteTitle, tags, createdAt, priority } = note;
```
Comment (MiheerTamkhane) [inline src/Component/EditorModal/EditorModal.jsx:8]: Nice approach destructuring makes code more readable.

File: src/Component/Sidenav/Navbar.jsx [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+          <NavLink
+            style={({ isActive }) => {
+              return {
+                color: isActive ? "purple" : "",
+              };
+            }}
+            to="/notes"
+            className="text-color-5  pd-y-5 text-dec text-s  "
+          >
+            {" "}
```
Comment (MiheerTamkhane) [inline src/Component/Sidenav/Navbar.jsx:79]: Remove this spaces

File: src/Pages/Notes/Notes.jsx [hunk 6]
```diff
@@ -1,9 +1,26 @@
 import React from "react";
+import CardNote from "../../Component/Card-note/CardNote";
+import NotesPanel from "../../Component/Note-Panel/notesPanel";
+import Navbar from "../../Component/Sidenav/Navbar";
```
Comment (MiheerTamkhane) [inline src/Pages/Notes/Notes.jsx:4]: Make barrel exporsts

**Review by MiheerTamkhane — APPROVED** [review]
The Preview of the PR is not working look on it. 
Other tha that good code structure and written also 
good to merge

File: src/Component/Card-note/CardNote.jsx (hunk 3 above)
Comment (saketkum11) [reply src/Component/Card-note/CardNote.jsx:30]: okk I forgot to do that

File: src/Component/Sidenav/Navbar.jsx (hunk 5 above)
Comment (saketkum11) [reply src/Component/Sidenav/Navbar.jsx:79]: Okk for sure

File: src/Pages/Notes/Notes.jsx (hunk 6 above)
Comment (saketkum11) [reply src/Pages/Notes/Notes.jsx:4]: yes .

**Pull request merged**